### PR TITLE
Start of work on draft -10.

### PR DESCRIPTION
**Pull request by jhaas-pfrc**
Put back AS_CONFED_SET.
Lots of word smithing.
Lots of xml and indentation cleanup.
Remove comments for things that have been addressed or otherwise accepted. Add two appendices: One covers the filtering example and moves the example out of the normative text. The second discusses inconsistent origin AS issues.

### CODE REVIEW

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (wkumari) [inline draft-ietf-idr-deprecate-as-set-confed-set.xml]: I'd suggest instead: 
"This document updates RFC 4271 and RFC 5065, and obsoletes RFC 6472." and then add "(if approved)" to the Updates: header.

The reason for this is that it would be embarrassing if we forgot this text in the final RFC version.

**Review by wkumari — APPROVED** [review]
I've made one (non-blocking / editorial) suggestion.

Because it's a minor note, I decided it was cleaner to just put it as a comment - feel free to address or not.

Thank you very much, kind sir...

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (jhaas-pfrc) [reply draft-ietf-idr-deprecate-as-set-confed-set.xml]: I've made the text change. (See latest diff.)
I'm not quite sure what you're asking about for the Updates: header.  The (if approved) text was present in that last diff.  See *10.txt which contains the compiled document.

I'll be waiting on approval from Sriram before doing merge and then let him do the publish to IETF.

Comment (wkumari) [reply draft-ietf-idr-deprecate-as-set-confed-set.xml]: Whoops, apologies, I hadn't checked the rendered output and had forgotten that XML2RFC / etc automagically add "(if approved)" (e.g: "Updates: 4271 5065 (if approved)").

So, we are just waiting on Sriram to submit I think...

**Comment by jhaas-pfrc** [comment]
Confirmed Sriram's desire to merge in email.

**Pull request merged**
